<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditAccount</class>
 <widget class="QWidget" name="EditAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelEditAccount">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Edit Account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttoNewAccount">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>New Account</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonExistAccount">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Existing Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCustomerID">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Customer ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSaldo">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Saldo</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAccountINFO">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Account information</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCustomerID">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSaldo">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditINFO">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>420</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSave">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBack">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>490</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCreditLimit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Credit limit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCreditLimit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>220</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>330</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDeleteAccount">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Delete Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAccountID">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Account ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAccountID">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>140</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
